globus.md

```diff
@@ -0,0 +1,88 @@
+---
+highlighter: none
+layout: default
+title: Using Globus to Transfer Files to and from CHTC
+---
+
+[Globus](https://www.globus.org/) is a data management service that lets you
+move files files between **endpoints**, computers that are connected to the
+Globus file transfer network.
+CHTC maintains a Globus endpoint that can serve as either the
+source or destination of a Globus file transfer
+(i.e., you can move your files out of CHTC to some other Globus endpoint,
+or from some other Globus endpoint to CHTC).
+
+All file transfer via Globus at CHTC is based out of the
+[`/staging`](/file-avail-largedata.shtml) directory.
+Therefore, to use Globus, you will need access to `/staging` 
+(instructions for requesting access are 
+[here](/file-avail-largedata.shtml#1-policies-and-intended-use)).
+
+
+## Using the CHTC Globus Endpoint
+
```

Comment:
What I'm missing here is "to transfer files using the CHTC globus endpoint, you can use the Globus web interface." 

Do people need to create an account on globus to use the web UI? If so, that should maybe be here or above under "requirements"

Reply:
Expanding the first comment above - I think you need an expository paragraph here describing: 
1) you'll be using the web interface
2) in the web interface, you'll select two end points (one will be CHTC)
3) choose files at one endpoint and transfer them to a folder on the other

And then continue with what you already have on how to do that.

Reply:
Added the expository paragraph, and notes about how to log in to the Globus web interface.